src/ldc/simd.di

```diff
@@ -204,15 +204,24 @@ assert(v.array == a);
 template loadUnaligned(V)
 if(is(typeof(llvmVecType!V)))
 {
-    alias BaseType!V T;
-    enum llvmT = llvmType!T;
+    alias BaseType!V ElementType;
+
+    pragma(inline, true)
+    deprecated("This is the DMD interface, use it only if cross-compiling. Otherwise, please use the LDC interface.")
```

Comment:
'Cross-compiling' usually means something completely different, compiling on a host for a different target. 'use it only for DMD compatibility' would be better IMO.

Reply:
Indeed. I meant to say "if you plan to compile with both DMD and LDC" but too versbose. "DMD compatibility" sells it better.

Reply:
Let's not add the deprecation message here. If you work on a codebase where deprecations == errors, you force people to immediately start using LDC's interface, while the DMD interface has not been deprecated by Dlang. Adding the deprecation here would mean that LDC would deprecate something that GDC and DMD don't; let's not fork things here :)

Reply:
Ok, it seems good to have it in a comment though. What do you think?

Reply:
Adding it to the documentation of the function is fine. `dmd -de ...` should work as before :)

Reply:
Isn't `-de` for features that are officially deprecated? i.e. we don't have control.
https://dlang.org/deprecate.html

Reply:
If you add `deprecated` calling the function will no longer compile with `-de`.

Reply:
Ah got it :)